File: app/src/main/java/com/inspiringteam/xchange/data/source/scopes/Local.java
package com.inspiringteam.xchange.data.source.scopes;

import java.lang.annotation.Documented;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import javax.inject.Qualifier;

/**
 * This scope has been created for Dagger to differentiate between types of Data Sources
 */

@Qualifier
@Documented
@Retention(RetentionPolicy.RUNTIME)
public @interface Local {

}

File: app/src/main/java/com/inspiringteam/xchange/data/source/scopes/Local.kt
package com.inspiringteam.xchange.data.source.scopes

import javax.inject.Qualifier

/**
 * This scope has been created for Dagger to differentiate between types of Data Sources
 */
@Qualifier
@MustBeDocumented
@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
annotation class Local 